eng/common/CIBuild.cmd

```diff
@@ -1,2 +1,2 @@
 @echo off
-powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0Build.ps1""" -restore -build -test -sign -pack -publish -ci %*"
\ No newline at end of file
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0Build.ps1""" -restore -build -test -integrationTest -sign -pack -publish -ci %*"
```

Comment:
I don't think we are supposed to change files under `eng/common/` as they will likely be overriden in the next arcade PR. Can we move this to build yml file instead?